GruntFile.js

```diff
@@ -57,7 +58,7 @@ module.exports = function(grunt) {
         bump: {
             options: {
                 files: bumpFiles,
-                commitFiles: bumpFiles,
+                commitFiles: bumpFiles.concat(['src/ChangeLog.html']),
```

Comment:
Not sure that I understand. Are you suggesting a change or just commenting? 

I just don't want to miss something.